linux_os/guide/services/ntp/chronyd_or_ntpd_specify_multiple_servers/rule.yml

```diff
@@ -37,8 +37,6 @@ rationale: |-
 
 severity: medium
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Containers have time services too. Reverting this change.

Reply:
ditto (see `chronyd_or_chronyc_network`)